src/operation/in_memory_model_store.jl

```diff
@@ -84,6 +84,18 @@ Base.isopen(store::InMemoryModelStore) = true
 Base.close(store::InMemoryModelStore) = nothing
 Base.flush(store::InMemoryModelStore) = nothing
 
+function Base.isempty(
+    store::InMemoryModelStore{T},
+) where {T <: AbstractModelOptimizerResults}
+    empty = true
+    for field in fieldnames(T)
+        value_container = getfield(store.data, field)
+        empty = isempty(value_container)
```

Comment:
This looks like a bug. The code returns the status of the last field. 
```
if !isempty(value_container)
    empty = false
end
```

Reply:
resolved